susemanager-utils/susemanager-sls/salt/certs/init.sls

```diff
@@ -13,7 +13,11 @@
 {{ includesls(grains['osfullname'], grains['osrelease_info']|first|string) }}
 {% endif -%}
 {% elif grains['os_family'] == 'RedHat' %}
+{% if grains['osfullname'] == 'Alibaba Cloud Linux (Aliyun Linux)' %}
```

Comment:
I guess you have a good reason, but I am curious now... what's the value of `os` and `osrelease`? I didn't have to do this for Amazon Linux or Alma.

Reply:
os:
Alibaba Cloud (Aliyun)
osfullname:
Alibaba Cloud Linux (Aliyun Linux)

Reply:
> Oh, and remember to fix the Java linting failures :-)

For consistency, I was not fixing those Java linting failures. Checkstyle was complaining about single-line methods but all the similar methods (right above the one I implemented) were also single-line.

Instead of suppressing the warnings, I just went an fixed the style on all those Java methods, even the ones that I have not implemented.

Reply:
> os:
> Alibaba Cloud (Aliyun)
> osfullname:
> Alibaba Cloud Linux (Aliyun Linux)

Mmmmm, to me the fact that `os` shows that, looks like a bug at saltstack. In all other cases we have, `os` is a "short name". But.. it's how it is :-)

> Instead of suppressing the warnings, I just went an fixed the style on all those Java methods, even the ones that I have not implemented.

On a separate PR? I guess you will need to rebase to this PR then?